src/loaders.py

```diff
@@ -142,10 +145,36 @@ def __getitem__(self, index):
         if self.annotations is not None:
             w, h = Xi.size
             yi = self.load_target(index, (h, w))
+
+            if self.augmenter:
+                image_augmented, boxes_augmented = self.augment(self.augmenter, Xi, yi[0])
+                Xi = image_augmented
+                yi = (boxes_augmented, yi[1])  # replace boxes with new boxes
+
             return Xi, yi
         else:
             return self.image_transform(Xi)
 
+    def augment(self, augmenter, image, boxes):
```

Comment:
@i008 There is a class that we are using to make imgaug a pytorch transform applicable. 
For example check https://github.com/neptune-ml/open-solution-salt-detection . 
I am ok with having it like this here though.

Reply:
Does pytorch transform API support bounding box transformations? I thought it does not so either way you can be able to mix them. But maybe i'm wrong did not check.

Reply:
It doesn't at the moment.

```python

class ImgAug:
    def __init__(self, augmenters):
        if not isinstance(augmenters, list):
            augmenters = [augmenters]
        self.augmenters = augmenters
        self.seq_det = None

    def _pre_call_hook(self):
        seq = iaa.Sequential(self.augmenters)
        seq = reseed(seq, deterministic=True)
        self.seq_det = seq

    def transform(self, *images):
        images = [self.seq_det.augment_image(image) for image in images]
        if len(images) == 1:
            return images[0]
        else:
            return images

    def __call__(self, *args):
        self._pre_call_hook()
        return self.transform(*args)

```

But creating a new one ImgAugBBoxes or something should be quite straightforward.